stream.go

```diff
@@ -435,6 +435,11 @@ func (ps *ProtoStream) Embedded(fieldNumber int, inner func(*ProtoStream) error)
 	return ps.writeAll(ps.childBuffer.Bytes())
 }
 
+// Write writes directly to the underlaying writer.
```

Comment:
"Write writes raw []byte to the underlying writer. It is the callers responsibility to make sure this wont yield a corrupt protobuf stream."

Reply:
Done in 5324d7c0b7a1f8fdabfe11780547f1bb4c2fe2fb 👍